instagram/src/components/CommentsSection/CommentsSection.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Comment from '../Comment/Comment';
+import PropTypes from 'prop-types';
+import './CommentsSection.css';
+import styled from 'styled-components';
+
+const Comments = styled.div`
+    width: 100%;
+    padding: 0 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+   
+    justify-content: flex-start;
+`;
+
+const CommentsSection = props => {
+    return (
+        <Comments>
+            {props.comments.map((comment, index) =>
+                <Comment 
+                    key={Math.random()}
```

Comment:
I think this is bad practice.

https://stackoverflow.com/questions/29808636/when-giving-unique-keys-to-components-is-it-okay-to-use-math-random-for-gener